src/driver/sqlserver/SqlServerQueryRunner.ts

```diff
@@ -4049,12 +4049,33 @@ export class SqlServerQueryRunner
             case "tinyint":
                 return this.driver.mssql.TinyInt
             case "char":
+                if (
+                    this.driver.options.options
+                        ?.disableAsciiToUnicodeParamConversion
+                ) {
+                    return this.driver.mssql.Char(...parameter.params)
+                }
+                return this.driver.mssql.NChar(...parameter.params)
             case "nchar":
                 return this.driver.mssql.NChar(...parameter.params)
```

Comment:
```suggestion
            case "char":
                if (
                    this.driver.options.options
                        ?.disableAsciiToUnicodeParamConversion
                ) {
                    return this.driver.mssql.Char(...parameter.params)
                }
            case "nchar":
                return this.driver.mssql.NChar(...parameter.params)
```
No idea if duplication of fall through is worse :D

Reply:
Thank you for the review! 

I'm happy to make the change, but question - the conditional in the "char" case is triggering "TS7029: Fallthrough case in switch" due to tsconfig.json's `"noFallthroughCasesInSwitch": true"`. Would you still like me to make the change, bypassing the error with a `// @ts-ignore`?

Reply:
as I said, both is not nice, just keep your version.